plugins/plugin-java/che-plugin-java-plain/che-plugin-java-plain-server/pom.xml

```diff
@@ -38,6 +42,14 @@
             <groupId>commons-io</groupId>
             <artifactId>commons-io</artifactId>
         </dependency>
+				<dependency>
```

Comment:
1. I don't have those tabs in my code... Looks like they were there for some time, but after I fixed them locally git ignores the fix. I'll try to fix
2. Probably because of:
```
+import javax.annotation.PostConstruct;
+import javax.annotation.PreDestroy;
```
in `PlainJavaProjectSourceFolderWatcher`